### PR TITLE
Adding afternoonlabs/oceanbreeze

### PR DESCRIPTION
**Pull request by eithanshavit**
## Description
Adding the new Ocean Breeze keyboard to Afternoon Labs

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by eithanshavit** [comment]
@fauxpark and @drashna, you both helped me review stuff before. Looking for some feedback. Thx!

**Review by fauxpark — CHANGES_REQUESTED** [review]
Looks good, except the info.json needs to be redone without the rotation:
image

**Comment by eithanshavit** [comment]
@fauxpark, I should probably know this by now, but how are you testing these info.json files? Where's the render you're showing from?

**Comment by fauxpark** [comment]
You can preview info.json files by pressing Ctrl+Shift+I in the Configurator.

**Comment by eithanshavit** [comment]
Thanks so much @fauxpark, I fixed the layout!

File: keyboards/afternoonlabs/oceanbreeze/rev1/info.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
+{
```
Comment (fauxpark) [inline keyboards/afternoonlabs/oceanbreeze/rev1/info.json:1]: Sorry, the thumb keys still seem to be slightly out of order:
image

**Comment by eithanshavit** [comment]
@fauxpark, I think I finally got it :) Sorry for the delay.

Screen Shot 2021-04-03 at 10 17 36 PM

**Comment by fauxpark** [comment]
Per the layout macro:
image

I think it should be more like this:
image

**Comment by eithanshavit** [comment]
Thanks @fauxpark, is there a requirement that the macro and the info.json will follow the same order? I think that my proposal right now makes sense from the eyes of a user of this keyboard. For the macro, it makes sense for someone configuring this layout visually per the macro layout. For the info.json layout and configurator, the current proposal makes sense because configuring the thumb clusters is a different process than the alphas. And so I don't necessarily agree with your latest screenshot of having the upper thumb keys be in order of the lower alpha row.

**Comment by fauxpark** [comment]
The info.json determines what order the keys will be placed into the top half of the layout macro, and so they must exactly match. You've placed LT4 and RT1 in the lower alpha row of the layout macro, which is about where they are physically, thus they have to be placed there in the info.json. If you want your ordering, you'll need to shift those four thumb keys down a line in the layout macro.

**Comment by eithanshavit** [comment]
I see, ok I've updated it to behave like this.
Screen Shot 2021-04-03 at 10 55 57 PM

**Comment by eithanshavit** [comment]
How's this looking now @fauxpark? Thanks for all the guidance

File: keyboards/afternoonlabs/oceanbreeze/rev1/rules.mk [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/afternoonlabs/oceanbreeze/rev1/rules.mk:23]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
SPLIT_KEYBOARD = yes
```

File: keyboards/afternoonlabs/oceanbreeze/rev1/rev1.h [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    {L35, L34, L33, L32, L31, L30, KC_NO, KC_NO, KC_NO}, \
+    {LT3, LT2, LT1, LT0, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO}, \
+    {LT4, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO}, \
+    {R00, R01, R02, R03, R04, R05, MC0, MC1, MC2}, \
+    {R10, R11, R12, R13, R14, R15, MC3, MC4, MC5}, \
+    {R20, R21, R22, R23, R24, R25, KC_NO, AUP, KC_NO}, \
+    {R30, R31, R32, R33, R34, R35, ALT, ADN, ART}, \
+    {RT2, RT3, RT4, RT5, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO}, \
+    {RT1, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO} \
+  }
```
Comment (fauxpark) [inline keyboards/afternoonlabs/oceanbreeze/rev1/rev1.h:43]:
```suggestion
#define XXX KC_NO

#define LAYOUT( \
    L00, L01, L02, L03, L04, L05,             R00, R01, R02, R03, R04, R05,   MC0, MC1, MC2, \
    L10, L11, L12, L13, L14, L15,             R10, R11, R12, R13, R14, R15,   MC3, MC4, MC5, \
    L20, L21, L22, L23, L24, L25,             R20, R21, R22, R23, R24, R25,        AUP, \
    L30, L31, L32, L33, L34, L35, LT4,   RT1, R30, R31, R32, R33, R34, R35,   ALT, ADN, ART, \
                   LT0, LT1, LT2, LT3,   RT2, RT3, RT4, RT5 \
) { \
    { L05, L04, L03, L02, L01, L00, XXX, XXX, XXX }, \
    { L15, L14, L13, L12, L11, L10, XXX, XXX, XXX }, \
    { L25, L24, L23, L22, L21, L20, XXX, XXX, XXX }, \
    { L35, L34, L33, L32, L31, L30, XXX, XXX, XXX }, \
    { LT3, LT2, LT1, LT0, XXX, XXX, XXX, XXX, XXX }, \
    { LT4, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX }, \
    { R00, R01, R02, R03, R04, R05, MC0, MC1, MC2 }, \
    { R10, R11, R12, R13, R14, R15, MC3, MC4, MC5 }, \
    { R20, R21, R22, R23, R24, R25, XXX, AUP, XXX }, \
    { R30, R31, R32, R33, R34, R35, ALT, ADN, ART }, \
    { RT2, RT3, RT4, RT5, XXX, XXX, XXX, XXX, XXX }, \
    { RT1, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX } \
}
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Looks pretty good, just a couple of minor cosmetic things.

**Review by fauxpark — APPROVED** [review]

**Comment by eithanshavit** [comment]
Looking for more reviews please 🙏

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**